pages/index.vue

```diff
@@ -221,38 +221,17 @@
 </template>
 
 <script>
-import { mapState, mapActions } from 'vuex'
+import { mapState, mapActions, mapGetters } from 'vuex'
 
 export default {
   head() {
     return {
       title: 'Home'
     }
   },
-  data() {
-    return {
-      loading: true,
-      coordinators: [
-        {
-          name: 'Aman Gupta',
-          avatar:
-            'https://lh6.googleusercontent.com/-Nabz6JJKsqc/AAAAAAAAAAI/AAAAAAAAAoI/biOxaDpjiec/photo.jpg'
-        },
-        {
-          name: 'Harshul Gupta',
-          avatar:
-            'https://res.cloudinary.com/amuroboclub/image/upload/v1529768915/profile_img/IMG_20180328_004220805.jpg'
-        },
-        {
-          name: 'Kakul Shrivastava',
-          avatar:
-            'https://res.cloudinary.com/amuroboclub/image/upload/v1543928884/profile_img/FB_IMG_1543928797162.jpg'
-        }
-      ]
-    }
-  },
   computed: {
-    ...mapState('news', ['news'])
+    ...mapState('news', ['news']),
+    ...mapGetters('team', ['members'])
```

Comment:
Instead of this make a different getter just for coordinators in index store. Currently complete team can be seen on the home page upon returning from team page. @iamareebjamal Your thoughts on this.

Reply:
How do I make a getter just for coordinators?

Reply:
Yup, just create a getter for coordinators. See how getters are made.